app/libs/classes/application_answer/zeep_person_info.py

```diff
@@ -0,0 +1,133 @@
+                        'ZIP': self._get_value(answers=applicantAnswers, tags=['postalCode']),
+                        'CITY': self._get_value(answers=applicantAnswers, tags=['postalAddress'])
+                    }
+                }
+            }
+        }
+
+    def _create_children(self):
+        child_answers: List[ApplicationAnswer] = self.application_answer_collection.filter_by_tags(
+            tags=[self.find_by_tag, 'children'])
+        if not child_answers:
+            return None
+
+        child = [self._create_child(answers)
+                 for answers in self._create_grouped_answers(child_answers)]
+
+        return {
+            self.person_type.upper(): {
+                'CHILD': child if len(child) > 1 else child[0],
+            },
+        }
+
+    def _create_grouped_answers(self, answers: List[ApplicationAnswer]) -> List[List[ApplicationAnswer]]:
+        grouped_answers: List[List[ApplicationAnswer]] = []
+
+        for answer in answers:
+            group_tag: str = answer.get_tag_starting_with('group:')
+            group_index = int(group_tag.split(':')[1]) if group_tag else 0
+
+            if len(grouped_answers) <= group_index:
+                grouped_answers.append([answer])
+            else:
+                grouped_answers[group_index].append(answer)
+
+        return grouped_answers
+
+    def _create_child(self, answers: List[ApplicationAnswer]) -> dict:
+        return {
+            'PNUMBER': self._get_value(answers, tags=['personalNumber']),
+            'FNAME': self._get_value(answers, tags=['firstName']),
+            'LNAME': self._get_value(answers, tags=['lastName']),
+            'ADDRESSES': {
+                'ADDRESS': {
+                    'TYPE': 'FB',
+                    'CO': '',
+                    'ADDRESS': self._get_value(answers, tags=['address']),
+                    'ZIP': self._get_value(answers, tags=['postalCode']),
+                    'CITY': self._get_value(answers, tags=['postalAddress'])
+                },
+            },
+            'PHONENUMBERS': {
+                'PHONENUMBER': {
+                    'TYPE':  self._get_value(answers, tags=['phoneType']),
+                    'NUMBER': self._get_value(answers, tags=['phoneNumber']),
+                    'SMS': False
+                },
+            },
+            'EMAIL': {
+                'EMAIL': '',
+                'NOTIFY': False
+            },
+            'FOREIGNCITIZEN': False,
+            'RESIDENCEPERMITTYPE': '',
+            'RESIDENCEPERMITDATE': '',
+            'CIVILSTATUS': '',
+            'ALTCIVILSTATUS': '',
+            'REGISTEREDATHOUSEHOLDADDRESS': '',
+            'ALTERNATELYWITHPARENTS': '',
+            'ISPARTTIMECHILD': '',
+            'PARTTIMECHILDDAYS': '',
+        }
+
+    def _get_value(self, answers: List[ApplicationAnswer], tags: List[str]) -> Union[str, int, float]:
+        search_tags = [*tags, self.person_type, self.find_by_tag]
```

Comment:
Seems like every use case for `_get_value` already uses a list filtered by `find_by_tag`, so we could probably remove that tag from here right?